corehq/apps/accounting/tasks.py

```diff
@@ -351,6 +355,41 @@ def send_purchase_receipt(payment_record, core_product, domain,
     )
 
 
+@task(ignore_result=True)
```

Comment:
may want to put this on the `queue=background_queue`. probably doesn't make sense to eat resources from tasks where users are actually waiting in the browser